README.md

```diff
@@ -46,6 +46,8 @@ A curated list of bitcoin services and tools for software developers
 * [Cashier-BTC](https://github.com/Overtorment/Cashier-BTC) - Self-hosted payment gateway.
 * [Chainradar API](https://github.com/yasaricli/chainradar-api) - Blockchain Explorer API for Chainradar.
 * [One-Time Address](https://github.com/alexk111/One-Time-Address) A better way to share your Bitcoin address.
+* [Messari.io](https://messari.io/api) JSON REST API (free as well as paid) with access to market data, news, metrics, profile, etc. 
```

Comment:
This is not blockchain data. Lets make a new section, something like "Market data API" and place it there

Reply:
Done.

See commit # 2 in 1f6847c73619257435efddb25e7e700c44a459b4